packages/ovm-toolchain/src/watcher.ts

```diff
@@ -27,12 +28,12 @@ export class Watcher {
     return this._getMessageHashesFromTx(false, l2TxHash)
   }
 
-  public onceL2Relay(msgHash: string, callback: Function): void {
-    return this._onceRelay(false, msgHash, callback)
+  public async getL2TransactionReceipt(l1ToL2MsgHash: string): Promise<any> {
```

Comment:
Probably worth adding the specific type (TransactionReceipt?) here to make people's auto-correct happy.